res/skins/QMLDemo/FadeBehavior.qml

```diff
@@ -0,0 +1,33 @@
+import QtQuick 2.12
+
+Behavior {
+    id: root
+
+    // Starting with QtQuick 2.15, we can use targetProperty.object here
+    property Item fadeTarget
+
+    SequentialAnimation {
+        NumberAnimation {
```

Comment:
assuming you don't need to access the opacity during the animation, you can try to use an [OpacityAnimator](https://doc.qt.io/qt-5/qml-qtquick-opacityanimator.html) instead. Not sure if that works when used as a value interceptor like in this case though...

Reply:
If you want to try that out, feel free to do that in a follow up. I struggled far too long with these animations (I initially planned to reanchor the cue/play button and the overview, too), but didn't manage to make it look good.